### DISCUSSION TITLE
Autodeleting Alias -- Time Based or Email Count Based

### DISCUSSION THREAD

**Discussion by TheMartianObserver** [anonaddy/anonaddy#227] [Feature Requests]
Love the service!

Was thinking that it might be nice to have a setting on aliases that allows for auto forget/delete after some threshold is met, such as number of emails received or some time has passed. Might be useful in cases where I need to provide an email address to sign up for something, but only need one email from the service.

Haven't fully thought through the UI implications...

Thanks!

**Comment by willbrowningme** [comment]
Interesting idea, will have a think about how this could work in practice.

**Comment by ririko5834** [comment]
Good idea

**Comment by http403** [comment]
It would be extra nice working with rules. Say mail prefixed with `burn_` would have receive one email then reject the rest.
